<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Depurador de Datos</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(58, 99, 140);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="dataSelectionContainer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>1031</width>
      <height>291</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 4px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="tableSelectionButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(58, 99, 140);
color: rgb(255, 255, 255);
border-radius: 4px;
</string>
     </property>
     <property name="text">
      <string>Seleccionar Tablas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileSelectionButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(58, 99, 140);
color: rgb(255, 255, 255);
border-radius: 4px;
</string>
     </property>
     <property name="text">
      <string>Seleccionar Archivos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="parameterSelectionButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(58, 99, 140);
color: rgb(255, 255, 255);
border-radius: 4px;
</string>
     </property>
     <property name="text">
      <string>Reglas de Negocio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyDepurationButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(58, 99, 140);
color: rgb(255, 255, 255);
border-radius: 4px;
</string>
     </property>
     <property name="text">
      <string>Aplicar Depuracion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downloadScriptButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(58, 99, 140);
color: rgb(255, 255, 255);
border-radius: 4px;
</string>
     </property>
     <property name="text">
      <string>Descargar</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="summayOfDataTable">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>541</width>
       <height>0</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 4px;
border-color:black;
</string>
     </property>
     <property name="midLineWidth">
      <number>-1</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>Tipo de Archivo</string>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo de Dato</string>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre de Archivo</string>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="summaryOfParameterTable">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>140</y>
       <width>641</width>
       <height>101</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Nombre Modificacion</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modificacion</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QLabel" name="dataSelectionWindowLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Seleccion de Datos</string>
    </property>
   </widget>
   <widget class="QFrame" name="actualDataContainer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>491</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 4px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="actualDataTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>411</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="newDataContainer">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>440</y>
      <width>511</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:4px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="newDataTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>431</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="actualDataLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Esquema Actual</string>
    </property>
   </widget>
   <widget class="QLabel" name="newDataLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>400</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Esquema Modificado</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
